snap/snapcraft.yaml

```diff
@@ -1,28 +1,26 @@
-name: charmed-mysql # you probably want to 'snapcraft register <name>'
-base: core22 # the base snap is the execution environment for this snap
-version: '8.0.37' # just for humans, typically '1.2+git' or '1.3.2'
-summary: MySQL server in a snap. # 79 char long summary
+name: charmed-mysql
+base: core22
+version: '8.0.37'
+summary: MySQL server in a snap.
 description: |
   The MySQL software delivers a very fast, multithreaded, multi-user,
   and robust SQL (Structured Query Language) database server. MySQL
   Server is intended for mission-critical, heavy-load production
   systems as well as for embedding into mass-deployed software.
 
-grade: stable # must be 'stable' to release into candidate/stable channels
-confinement: strict # use 'strict' once you have the right plugs and slots
+architectures:
+  - build-on: [amd64]
+    build-for: [amd64]
+  - build-on: [arm64]
+    build-for: [arm64]
+grade: stable
+confinement: strict
 license: Apache-2.0
 
 system-usernames:
   snap_daemon: shared
 
 package-repositories:
-  - type: apt
-    components:
-      - main
-    suites:
-      - jammy
-    key-id: F6ECB3762474EDA9D21B7022871920D1991BC93C
-    url: http://security.ubuntu.com/ubuntu/
```

Comment:
i believe we used this package repository to get packages (mysql-server and mysql-router) with security updates. any particular reason why this was removed?

Reply:
As discussed in private, this is no longer necessary (created in a moment of pinning 8.0.32).
Security repos is enabled by default, no need to add in yaml manually.

Reply:
if i can remember correctly, this entry ensures that we get packages from the security pocket (please [see this post](https://askubuntu.com/questions/1279002/what-is-the-difference-between-suite-and-suite-security-repositories-in-apt) for an explanation i went off of). it seems to imply that release pockets do not get security updates from the canonical security team